### PR TITLE
Upgrading to ember 2.5.0

### PR DESCRIPTION
**Pull request by jseros**
- It also bumps up to 0.2.0
- Removes several deprecation notices

### CODE REVIEW

**Comment by jseros** [comment]
It failed for ember 1.13 as expected. Should this addon continue supporting 1.13?

**Comment by josemarluedke** [comment]
@jseros Thank you for putting your time on this! Yes, let's drop 1.13 support. Could you remove it from Travis?

File: bower.json.ember-try [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "ember-cli-segment",
+  "dependencies": {
+    "jquery": "1.11.3",
+    "ember": "1.13.13",
```
Comment (josemarluedke) [inline bower.json.ember-try]: Shouldn't this be 2.5.0?

File: tests/dummy/app/index.html [hunk 2]
```diff
@@ -7,27 +7,19 @@
...
     <link rel="stylesheet" href="assets/dummy.css">
 
-    {{content-for 'head-footer'}}
+    {{content-for "head-footer"}}
   </head>
   <body>
-    {{content-for 'body'}}
+    {{content-for "body"}}
 
-    <script>
```
Comment (josemarluedke) [inline tests/dummy/app/index.html]: We need to keep this.

Comment (jseros) [inline tests/dummy/app/index.html]: Oh! my bad! fixing it now

**Comment by jseros** [comment]
Done!

**Comment by josemarluedke** [comment]
Thank you @jseros!

**Comment by jseros** [comment]
Thank you! Can you please launch a release so that I can update my project? Thanks again!

**Comment by josemarluedke** [comment]
@jseros It's already done!

**Pull request merged**